querier.py

```diff
@@ -1,229 +1,34 @@
-"""import statements"""
```

Comment:
Same as in other scripts: hard to review. What has really changed with respect to the original code?